hw1/02-palindrome.js

```diff
@@ -1,3 +1,25 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  const value = parseFloat(elem.value);
+  const resultDiv = document.querySelector("#result");
+
+  if (value >= 0) {
+    const numString = value.toString();
+    const reverseString = numString.split("").reverse().join("");
```

Comment:
logic of checking palindrome can be put into a seperate simple funtion for easy code review.

Reply:
Gotcha, this makes the code much cleaner. I'll go ahead and update that. Thanks!